test/regression.uts

```diff
@@ -1819,6 +1819,10 @@ pktpcapngdefaults = rdpcap(pcapngdefaults)
 assert pktpcapngdefaults[0].time == 1575115986.114775512
 assert Ether in pktpcapngdefaults[0]
 
+= Read a pcapng with little-endian SHB
+pktcapng = sniff(offline=scapy_path("/test/pcaps/macos.pcapng"))
```

Comment:
Is it possible to reduce the size of this pcap?

Reply:
Done. Remember to squash on merge for this to actually make sense